File: backend/adapters/src/main/java/br/com/unifor/adapters/dto/AlunoDTO.java
package br.com.unifor.adapters.dto;

import lombok.*;
import lombok.experimental.SuperBuilder;
import java.time.LocalDate;

@Getter
@Setter
@SuperBuilder
@NoArgsConstructor
@AllArgsConstructor
public class AlunoDTO extends AuditableDTO {
    private String nome;
    private String email;
    private LocalDate dataNascimento;
    private UsuarioDTO usuario;
    private CursoDTO curso;
}

File: frontend/unifor-frontend/src/app/features/admin/pages/alunos/aluno.dto.ts
import { UsuarioDTO } from '../usuario/usuario.dto';
import {CursosDTO} from "../cursos/cursos.dto"

export interface AlunoDTO {
  identifier?: number;
  nome: string;
  email: string;
  dataNascimento: string;
  usuario: UsuarioDTO;
  curso?: CursosDTO;
}
